wasm/richards/benchmark.js

```diff
@@ -0,0 +1,27 @@
+// Copyright 2025 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+class Benchmark {
+  async runIteration() {
+    // Instantiate the Wasm module before the first run.
+    if (!Module._setup)
+      await setupModule(Module);
+
+    // Set-up the problem (fill the work queue) on each run.
+    Module._setup();
+
+    // Repeatedly call into Wasm to stress test JS-to-Wasm call performance.
+    // I (dlehmann) suppose this wrapper was added (originally in JetStream 2)
+    // to test that a wrapping JavaScript function doesn't slow things down.
```

Comment:
Mild preference for removing this wrapper function and calling into Wasm directly?

Reply:
Sure, I am happy either way, if everyone agrees (I went for maximum consistency with the previous version as a first approach).

Reply:
Don't recall the context but it was probably added to stress JS->Wasm bridging as that was something we thought / still think is important. I think the assumption is that a reasonable number of applications us the wasm host to drive their workloads. So, I'd prefer we leave it for now and reconsider later.

Reply:
Then let's leave it as-is. Jan has voiced only a slight preference, and at least in practical terms, it doesn't seem to matter with Spidermonkey, JSC, or V8 right now (no measurable difference, at least V8 definitely inlines the JS wrapper anyway).